{"uid":"ad2afe52980f4a40","name":"Проверка возможности подписаться","fullName":"tests.test_profile.TestProfile#test_opportunities_to_subscribe","historyId":"8837be03da4b857171429e7b0d59755b","time":{"start":1739379810847,"stop":1739379810847,"duration":0},"description":"После проверки, отписываемся от пользователя для приведения в исходное состояние","descriptionHtml":"<p>После проверки, отписываемся от пользователя для приведения в исходное состояние</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message:","statusTrace":"request = <SubRequest 'login_to_app' for <Function test_opportunities_to_subscribe>>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"814405627e4dcb68af55f22c6bbee665\")>\n\n    @pytest.fixture(scope='function')\n    @allure.title('Авторизация пользователя в системе')\n    @allure.step('Авторизация пользователя в системе')\n    def login_to_app(request, driver):\n        \"\"\" Фикстура Логина в систему \"\"\"\n        login_page = ApiAuthorization(driver)\n        login_page.user(User.LOGIN, User.PASSWORD)\n        try:\n            later = login_page.find_element(config.locators.RubricPageLocators.locators['Кнопка выберу потом'])\n            login_page.do_click(later)\n        except TimeoutException:\n            element = login_page.find_element(\n                config.locators.MainPageLocators.locators['Листай вниз и найди свои интересы'])\n            login_page.do_click(element)\n        finally:\n>           login_page.do_click(config.locators.MainPageLocators.locators['Листай вниз и найди свои интересы'])\n\nconftest.py:62: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nbase/base_page.py:80: in do_click\n    elem = WebDriverWait(self.driver, wait_time).until(EC.element_to_be_clickable(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"814405627e4dcb68af55f22c6bbee665\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f222f4a1b20>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n../local/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1739378850849,"stop":1739378850850,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1739379810848,"stop":1739379811275,"duration":427},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1739379811275,"stop":1739379811276,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Инициализация локаторов","time":{"start":1739379811276,"stop":1739379811276,"duration":0},"status":"passed","steps":[{"name":"Инициализация локаторов","time":{"start":1739379811276,"stop":1739379811276,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"request","value":"<SubRequest 'elements' for <Function test_opportunities_to_subscribe>>"}],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false},{"name":"Авторизация пользователя в системе","time":{"start":1739379811276,"stop":1739379834908,"duration":23632},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \n\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/conftest.py\", line 62, in login_to_app\n    login_page.do_click(config.locators.MainPageLocators.locators['Листай вниз и найди свои интересы'])\n  File \"/usr/workspace/base/base_page.py\", line 80, in do_click\n    elem = WebDriverWait(self.driver, wait_time).until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Авторизация пользователя в системе","time":{"start":1739379811276,"stop":1739379834907,"duration":23631},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \n\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/conftest.py\", line 62, in login_to_app\n    login_page.do_click(config.locators.MainPageLocators.locators['Листай вниз и найди свои интересы'])\n  File \"/usr/workspace/base/base_page.py\", line 80, in do_click\n    elem = WebDriverWait(self.driver, wait_time).until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[{"name":"request","value":"<SubRequest 'login_to_app' for <Function test_opportunities_to_subscribe>>"},{"name":"driver","value":"<selenium.webdriver.chrome.webdriver.WebDriver (session=\"814405627e4dcb68af55f22c6bbee665\")>"}],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"driver::0","time":{"start":1739379834919,"stop":1739379835125,"duration":206},"status":"passed","steps":[],"attachments":[{"uid":"904b947aa6ecb0d8","name":"page_source","source":"904b947aa6ecb0d8.html","type":"text/html","size":35614},{"uid":"688bd8c66186478e","name":"screenshot","source":"688bd8c66186478e.png","type":"image/png","size":158730},{"uid":"650b3f0c0f49610b","name":"browser_logs","source":"650b3f0c0f49610b.txt","type":"text/plain","size":3727}],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":3,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"labels":[{"name":"feature","value":"Профиль пользователя"},{"name":"severity","value":"Critical"},{"name":"tag","value":"regression"},{"name":"tag","value":"profile"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_profile"},{"name":"subSuite","value":"TestProfile"},{"name":"host","value":"83c52f07f834"},{"name":"thread","value":"1-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_profile"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":13,"broken":12,"skipped":0,"passed":19,"unknown":0,"total":44},"items":[{"uid":"d5a549eef9d9896","status":"failed","statusDetails":"AssertionError: Ожидался текст: '4', отображается текст: '3'","time":{"start":1739310776787,"stop":1739310779760,"duration":2973}},{"uid":"5416cd93f38ba8a3","status":"broken","statusDetails":"selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: element is not attached to the page document\n  (Session info: headless chrome=112.0.5615.165); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception","time":{"start":1738952232456,"stop":1738952233625,"duration":1169}},{"uid":"1ebf252b33a6b2c5","status":"broken","statusDetails":"KeyError: 'access_token'","time":{"start":1738875260341,"stop":1738875260341,"duration":0}},{"uid":"47c5616a81a9004f","status":"broken","statusDetails":"selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: element is not attached to the page document\n  (Session info: headless chrome=112.0.5615.165); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception","time":{"start":1738872432097,"stop":1738872433761,"duration":1664}},{"uid":"e0eaddd146eee04a","status":"broken","statusDetails":"KeyError: 'access_token'","time":{"start":1738865312078,"stop":1738865312078,"duration":0}},{"uid":"83a1d83d3050f111","status":"failed","statusDetails":"AssertionError: Ожидался текст: '2', отображается текст: '1'","time":{"start":1738269328664,"stop":1738269334101,"duration":5437}},{"uid":"393fc5fb81706fa9","status":"failed","statusDetails":"AssertionError: Ожидался текст: '2', отображается текст: '1'","time":{"start":1738099800199,"stop":1738099808600,"duration":8401}},{"uid":"1a1c26198a21bd47","status":"failed","statusDetails":"AssertionError: Ожидался текст: '2', отображается текст: '1'","time":{"start":1738005655996,"stop":1738005661415,"duration":5419}},{"uid":"bb37ea67897004f9","status":"failed","statusDetails":"AssertionError: Ожидался текст: '2', отображается текст: '1'","time":{"start":1738002934635,"stop":1738002940052,"duration":5417}},{"uid":"41baf7984698b959","status":"broken","statusDetails":"selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\n  (Session info: chrome=132.0.6834.83); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception","time":{"start":1737932558826,"stop":1737932558826,"duration":0}},{"uid":"c64e506b5b3bf6fd","status":"failed","statusDetails":"AssertionError: Ожидался текст: '2', отображается текст: '1'","time":{"start":1737843315212,"stop":1737843322169,"duration":6957}},{"uid":"44c71df5ac64d686","status":"failed","statusDetails":"AssertionError: Ожидался текст: '2', отображается текст: '1'","time":{"start":1737832501768,"stop":1737832508742,"duration":6974}},{"uid":"ce1adce00b0b9cd","status":"failed","statusDetails":"AssertionError: Ожидался текст: '2', отображается текст: '1'","time":{"start":1737799865822,"stop":1737799872800,"duration":6978}},{"uid":"c3ba876983d362af","status":"failed","statusDetails":"AssertionError: Ожидался текст: '2', отображается текст: '1'","time":{"start":1737758099860,"stop":1737758106847,"duration":6987}},{"uid":"681cab165fc587d7","status":"failed","statusDetails":"AssertionError: Ожидался текст: '2', отображается текст: '1'","time":{"start":1737745092073,"stop":1737745099043,"duration":6970}},{"uid":"6412b3cf34021890","status":"failed","statusDetails":"AssertionError: Ожидался текст: '2', отображается текст: '1'","time":{"start":1737652984595,"stop":1737652992082,"duration":7487}},{"uid":"6f9148b3096adb32","status":"failed","statusDetails":"AssertionError: Ожидался текст: '2', отображается текст: '1'","time":{"start":1737570658842,"stop":1737570666395,"duration":7553}},{"uid":"1a48054d733defb1","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium-browser is no longer running, so ChromeDriver is assuming that Chrome has crashed.)","time":{"start":1737568968602,"stop":1737568968602,"duration":0}},{"uid":"5f1fbecb0e9504cc","status":"failed","statusDetails":"AssertionError: Ожидался текст: '8', отображается текст: '7'","time":{"start":1734034910933,"stop":1734034917944,"duration":7011}},{"uid":"6c8e11255c7cf244","status":"passed","time":{"start":1733937646919,"stop":1733937656990,"duration":10071}}]},"tags":["regression","profile"]},"source":"ad2afe52980f4a40.json","parameterValues":[]}